libstuff/libstuff.h

```diff
@@ -772,11 +772,10 @@ void SQueryLogOpen(const string& logFilename);
 void SQueryLogClose();
 
 // Returns an SQLite result code.
-int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result,
-           int64_t warnThreshold = 2000 * STIME_US_PER_MS, bool skipWarn = false);
-inline int SQuery(sqlite3* db, const char* e, const string& sql, int64_t warnThreshold = 2000 * STIME_US_PER_MS, bool skipWarn = false) {
+int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, bool skipWarn = false);
+inline int SQuery(sqlite3* db, const char* e, const string& sql, bool skipWarn = false) {
```

Comment:
Actually, one small change, drop the `inline` here, we've stopped using it and let the compiler decide on it's own optimizations. I'd actually put the implementation in `SQLite.cpp`, too, but that's minor.

Reply:
I don't know why, but I get errors if I drop the `inline`
maybe I should do something else, like changing all the SQuery calls from SQLite?

Reply:
I would move the method definition into the cpp file, so you have only:
```
int SQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, bool skipWarn = false);
int SQuery(sqlite3* db, const char* e, const string& sql, bool skipWarn = false);
```

In SQLite.h, and the definitions for both functions on SQLite.cpp.